daft/connect/__init__.py

```diff
@@ -0,0 +1,115 @@
+"""Daft Connect provides a simple way to start a Spark server and connect to it from a client.
+
+Additionally, Daft Connect provides a Ray remote class that wraps the the daft.daft.connect_start function.
+
+Usage:
+```py
+from daft.daft import connect_start
+
+connect_server = connect_start()  # optionally pass in a port `connect_start(port=1234)`
+
+# Get the url to connect to the spark service
+port = connect_server.port()
+url = f"sc://localhost:{port}"
```

Comment:
Should this be available on `connect_server.url()`? In the future I guess this might not be localhost